Homework_2/HW_2_Q_2.java

```diff
@@ -0,0 +1,28 @@
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HW_2_Q_2 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader buf = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("Enter the string : ");
+        String s = buf.readLine();
+
+        Map<Character, Integer> mp = new HashMap<Character, Integer>();
```

Comment:
Use a name for the map that indicates the mapping like frequencyOfChar or charToFrequency